app/models/recipe.rb

```diff
@@ -1,3 +1,13 @@
 class Recipe < ApplicationRecord
-   validates_presence_of :name, :complexity, :genre
+  has_many :recipe_ingredients
+  has_many :ingredients, through: :recipe_ingredients
+  validates_presence_of :name, :complexity, :genre
+
+  def find_ingredients
```

Comment:
Great job making a method here but I would probably just iterate over my `recipe.ingredients` in the view.